Objects/object_layout.md

```diff
@@ -7,14 +7,14 @@ Each Python object starts with two fields:
 * ob_refcnt
 * ob_type
 
-which the form the header common to all Python objects, for all versions,
+which form the header common to all Python objects, for all versions,
 and hold the reference count and class of the object, respectively.
 
 ## Pre-header
 
 Since the introduction of the cycle GC, there has also been a pre-header.
 Before 3.11, this pre-header was two words in size.
-It should be considered opaque to all code except the cycle GC.
+It should be considered opaque to all of the code except the cycle GC.
```

Comment:
This was ok and I do not see this as improvement, though someone else might disagree.
```suggestion
It should be considered opaque to all code except the cycle GC.
```